util/regexp-assemble/data/934100.data

```diff
@@ -24,9 +24,150 @@
 
 _\$\$ND_FUNC\$\$_
 __js_function
-\beval\s*\(
-String\s*\.\s*fromCharCode
-function\s*\(\s*\)\s*{
+\beval\(
+String\.fromCharCode
+function\(\){
 new\s+Function\s*\(
 this\.constructor
-module\.exports\s*=
+module\.exports=
+
+
+##!> assemble
+\[[\'\"\`]
+##!=< js-prop-start
+##!<
+
+##!> assemble
+[\'\"\`]\]
+##!=< js-prop-finish
+##!<
+
+##!> assemble
+\(
+\.call\(
+##!=< js-call
+##!<
+
+
+##! "process" payloads
+##!> assemble
+process
+##!=>
+
+##! All method names
+##!> assemble
```

Comment:
Please indent nested processors to improve readability (will not repeat for the same issues below)